src/index.js

```diff
@@ -7,6 +7,8 @@ import { updateLikes } from './module/aboutLikes.js';
 
 displayList();
 addHeaderToPage();
+updateLikes();
+// countItems();
```

Comment:
Please consider not committing the comments because it makes your code look messy

Reply:
As suggested, I have removed the comments from the code. Thank you